src/actions.js

```diff
@@ -0,0 +1,45 @@
+// Action types in game.
+export const CLICK_CELL = 'CLICK_CELL';
+export const CHANGE_GAME_INDEX = 'GAME_INDEX';
+export const RESET_GAME = 'RESET_GAME';
+export const CHANGE_BOT = 'CHANGE_BOT';
+export const BOT_RUNNING = 'BOT_RUNNING';
```

Comment:
I tend to make all of my action types verbs, so in this case you might go with `SET_BOT_RUNNING`.